### PR TITLE
[Deploy] update HF template

### PR DESCRIPTION
**Pull request by fedml-zijianhu**

### CODE REVIEW

File: python/fedml/api/modules/model.py [hunk 1]
```diff
@@ -44,24 +44,28 @@ def create(name: str, model: str = None, model_config: str = None) -> bool:
 
 
 def create_from_hf(name: str, model: str = None) -> bool:
-    # Copy the template folder to the current folder
     hf_templ_fd_src = os.path.join(os.path.dirname(__file__), "..", "..", "serving", "templates", "hf_template")
-    # Copy recursively
-    dst_fd_prefix = "temp_"
-    dst_fd = os.path.join(os.getcwd(), dst_fd_prefix + name)
+    dst_parent_fd = os.path.join(os.path.expanduser("~"), ".fedml","fedml-model-client", "fedml", "hf_model_from_template")
```
Comment (fedml-zijianhu) [inline python/fedml/api/modules/model.py:48]: What happens if the user wants to create multiple HF model cards? Is the model card file created from template copied in another function?

Comment (Raphael-Jin) [reply python/fedml/api/modules/model.py:48]: `dst_fd = os.path.join(dst_parent_fd, name)`

Comment (Raphael-Jin) [reply python/fedml/api/modules/model.py:48]: I think Line 52:

Comment (Raphael-Jin) [reply python/fedml/api/modules/model.py:48]: The local model card folder, says `dst_fd`, was a copy of `python/fedml/serving/templates/hf_template/` 

According to Line 52:
`dst_fd = os.path.join(dst_parent_fd, name)`
the `name` here means the model card name. So there could be multiple local model card folder with different names that created (copied) from the template folder.

Comment (fedml-zijianhu) [reply python/fedml/api/modules/model.py:48]: Got it. Looks good to me

**Pull request merged**